cmd/zed/create/command.go

```diff
@@ -48,9 +50,10 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 		seekStride: units.Bytes(data.DefaultSeekStride),
 	}
 	c.thresh = data.DefaultThreshold
+	f.Var(&c.seekStride, "seekstride", "size of seek-index unit for ZNG data, as '32KB', '1MB', etc.")
 	f.Var(&c.thresh, "S", "target size of pool data objects, as '10MB' or '4GiB', etc.")
```

Comment:
Nit: Keep the c.thresh assignment adjacent to the corresponding f.Var call.
```suggestion
	f.Var(&c.seekStride, "seekstride", "size of seek-index unit for ZNG data, as '32KB', '1MB', etc.")
	c.thresh = data.DefaultThreshold
	f.Var(&c.thresh, "S", "target size of pool data objects, as '10MB' or '4GiB', etc.")
```